### ISSUE TITLE
Type error with 'options' in Text

### ISSUE THREAD

**Issue by CKylinMC** [kitschpatrol/svelte-tweakpane-ui#17]
Svelte Tweakpane UI version: 1.3.0
Svelte version: 4.2.19

![image](https://github.com/user-attachments/assets/6eb1f267-f525-4745-8c72-f2c5ef98523f)
I'm trying to force this text field to be "text", and I found that I can add "`{view: 'string'}`" into options to do that. It works, but one issue is, it will always throwing an error from typescript tell me "no 'options' here". 

And I see through the .d.ts file and I found it has been excluded by definition.
![image](https://github.com/user-attachments/assets/5b905537-4d46-4431-8cba-77cdc4681bbc)

Did this property blocked for any reason?

**Comment by kitschpatrol** [comment]
Hi, thanks for for using the library and opening an issue.

I might have just missed this prop or not seen a use for it that wouldn't already be covered by the [`<Color>`](https://kitschpatrol.com/svelte-tweakpane-ui/docs/components/color) component? (And just as a general tip, both auto-completions in your editor and [the component documentation pages](https://kitschpatrol.com/svelte-tweakpane-ui/docs/components/text) should provide a correct list of supported props.)

Can you explain what you're trying to do at a higher level? I'm guessing you have a hex string that you want to edit directly as text, and Tweakpane is giving you the color control instead?

From a quick look at this, it just seems like an oversight on my part and I should have hard-coded `view` to `text` in the internal implementation of the [`<Text>`](https://kitschpatrol.com/svelte-tweakpane-ui/docs/components/text) component,  and in the documentation I should encourage the more semantically explicit use of the [`<Color>`](https://kitschpatrol.com/svelte-tweakpane-ui/docs/components/color) component when a color control is desired.

**Comment by kitschpatrol** [comment]
Thinking about this more, it's basically a bug... the `<Text>` component should _always_ give you a control with `view: 'text'` set internally, regardless of the semantic content of `value`.

I've addressed this in [v1.3.2](https://github.com/kitschpatrol/svelte-tweakpane-ui/releases/tag/v1.3.2).

Thanks for bringing it to my attention.

Note that Tweakpane's `view` option is still intentionally not exposed, since Svelte Tweakpane UI's separates Tweakpane control types into discrete components instead of inferring control types from values. (With a few exceptions for the lower-level component.)

**Comment by CKylinMC** [comment]
Yes, I want it force to be text just because it always turn my hex-like string into color selector, and I see that used to be an issue in tweakpane v2/v3 and after that they use "`{view: 'string'}`" to control it, so I tried to add it manually.

Since this issue has been solved internally, I can let the 'options' go. Thanks!

*Btw I have another issue with `<Element>`, when I use it in "if" block, it sometimes make the next blade fly away. I'm still finding what cause that. For reproduce, see [this](https://svelte.dev/repl/c6fe4cc94d894c90bac22d5bc7dc56a3?version=4.2.19)*
